unit-test/ui/models/mixins/normalize-company-name.es6.js

```diff
@@ -0,0 +1,39 @@
+let normalizeCompanyName = require('../../../../shared/js/ui/models/mixins/normalize-company-name.es6')
+normalizeCompanyName = normalizeCompanyName.normalizeCompanyName
```

Comment:
I wanted to do `require('...etc/etc/etc')(normalizeCompanyName)` but then it wouldn't recognize normalizeCompanyName() as a function, so I had to do this on 2 lines.